api/server/server.go

```diff
@@ -468,6 +468,12 @@ func WithAgentFromEnv() ServerOption {
 				placer = pool.NewNaivePlacer()
 			}
 
+			// If prometheus is enabled, add LB placer metrics to the views
+			if s.promExporter != nil {
```

Comment:
don't think we should be doing this as a general rule of thumb -- which exporters we have attached shouldn't affect which metrics we collect (easy to fix...) -- if we need a way to turn these off we kinda have it half plumbed with the register views methods throughout now, seems like we could do similar to as was done in #1057 to have the default set of views and let the user either import the package that turns on all our views or they can add them individually themselves. anyway, not too big a deal

Reply:
@rdallman thanks for pointing this out, saw your PR and just merged it. :-)